src/compiler/compile/nodes/EventHandler.ts

```diff
@@ -17,7 +17,7 @@ export default class EventHandler extends Node {
 	constructor(component: Component, parent, template_scope, info) {
 		super(component, parent, template_scope, info);
 
-		this.name = info.name;
+		this.name = info.name !== '*' ? info.name : 'any';
```

Comment:
This would give `on:any` a different meaning than it already has, which we don't want to do. It's possible that someone has code where they are bubbling an event called `any`, and this would break it. I think elsewhere should just be looking for `handler.name === '*'` directly.

Reply:
Renaming to `$$any` helped. Is it ok way this way?